regolith/helpers/l_milestoneshelper.py

```diff
@@ -34,6 +35,9 @@ def subparser(subpi):
                             f" Default is active projecta, i.e. 'started'",
                        default=None
                        )
+    subpi.add_argument("-f", "--find", nargs="+",
```

Comment:
make sure you add this to the list of args that should not be deleted (i.e., --date) so when people use another one as template they don't delete it.

Also, maybe ``find`` -> ``filter`` here